src/server/pkg/deploy/cmds/cmds.go

```diff
@@ -524,6 +526,7 @@ particular backend, run "pachctl deploy storage <backend>"`,
 	deploy.PersistentFlags().IntVar(&pachdShards, "shards", 16, "(rarely set) The maximum number of pachd nodes allowed in the cluster; increasing this number blindly can result in degraded performance.")
 	deploy.PersistentFlags().IntVar(&etcdNodes, "dynamic-etcd-nodes", 0, "Deploy etcd as a StatefulSet with the given number of pods.  The persistent volumes used by these pods are provisioned dynamically.  Note that StatefulSet is currently a beta kubernetes feature, which might be unavailable in older versions of kubernetes.")
 	deploy.PersistentFlags().StringVar(&etcdVolume, "static-etcd-volume", "", "Deploy etcd as a ReplicationController with one pod.  The pod uses the given persistent volume.")
+	deploy.PersistentFlags().StringVar(&etcdStorageClassName, "etcd-storage-class-name", "", "If set, the name of an existing StorageClass to use for etcd storage. Ignored if --static-etcd-volume is set.")
```

Comment:
No problem - Done!

Reply:
I retained the `Name` suffix on all variable names, since some of them pre-dated this PR. I'm happy to swap these also, if you like, it just seemed a little noisy.